tests/test_routes.py

```diff
@@ -0,0 +1,99 @@
+from app.models.planet import Planet
+
+def create_mock_data(client):
+    a = {
+        "name": "foo1",
+        "color": "bar1",
+        "description": "foobar1"
+    }
+
+    b = {
+        "name": "foo2",
+        "color": "bar2",
+        "description": "foobar2"
+    }
+
+    c = {
+        "name": "foo3",
+        "color": "bar3",
+        "description": "foobar3"
+    }
+
+    client.post("/planets", json=a)
+    client.post("/planets", json=b)
+    client.post("/planets", json=c)
+
+def test_get_all_planets_returns_empty_list_when_db_is_empty(client):
+    #act
+    response = client.get("/planets")
+
+    #assert
+    assert response.status_code == 200
+    assert response.get_json() == []
+
+def test_get_all_planets(client):
+    #act
+    create_mock_data(client)
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    #assert
+    assert response.status_code == 200
+    assert len(response_body) == 3
+    assert response_body[0]["name"] == "foo1"
+    assert response_body[1]["name"] == "foo2"
+    assert response_body[2]["name"] == "foo3"
+    assert response_body[0]["color"] == "bar1"
+    assert response_body[1]["color"] == "bar2"
+    assert response_body[2]["color"] == "bar3"
+    assert response_body[0]["description"] == "foobar1"
+    assert response_body[1]["description"] == "foobar2"
+    assert response_body[2]["description"] == "foobar3"
```

Comment:
You could also do something like:

```suggestion
     assert response_body == [ {
         "name": "foo1",
         "color": "bar1",
         "description": "foobar1",
       },
       {
         "name": "foo2",
         "color": "bar2",
         "description": "foobar2",
       }
       ...
   ```
```